ui/src/App.js

```diff
@@ -69,12 +69,17 @@ class App extends Component {
             if (field.description) {
               return {
                 label: field.name + " - " + field.description,
-                value: field.elasticsearch_name
+                value: field.elasticsearch_name,
+                chipLabel: (field.name + " - " + field.description).substr(
```

Comment:
nit: Could extract `field.name + " - " + field.description` to local var since it's used twice